rstuf.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import json
+
+RSTUF_API_BASE_URL = "http://rstuf.local/api/v1"
```

Comment:
This one should be read as, say, an environment variable so that it's that more easily configurable.